src/main.js

```diff
@@ -12,6 +12,13 @@ export default function (Vue, { router, head, isClient }) {
       color: white;
       `
   }
+  router.beforeEach((to, from, next) => {
+    if (to.name === '*') {
+      router.push('/404');
+    } else {
+      next();
+    }
+  });
```

Comment:
I think gridsome should handle the redirect for us as long as we have `src/pages/404.vue`. Was this for something else?

Reply:
My bad, I didn't know the redirect was already handled